### PR TITLE
started backend

### PR DESCRIPTION
**Pull request by jefcolbi**

### CODE REVIEW

File: backend/camerapps/settings.py [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/3.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-&i1!9_xfz&icb6-!yd52)6e6)ru8&*8%_g#*nhr^i0!k_183r='
```
Comment (abdounasser202) [inline backend/camerapps/settings.py:23]: SECRET_KEY should be in a .env file

**Review by abdounasser202 — CHANGES_REQUESTED** [review]

**Comment by elhmn** [comment]
I am closing this as it does not really fit with the plan we have for this project at the moment. @jefcolbi thanks a lot for this contribution but it was unfortunately not very aligned with the roadmap 🙇‍♂️

**Comment by abdounasser202** [comment]
Can we please have the roadmap of this project ?

**Comment by elhmn** [comment]
> Can we please have the roadmap of this project ?

As it has been discussed several times in the chat that this project is not supposed to receive any major improvement as another version is currently being built on this repository https://github.com/osscameroon/camerapps. I am currently unaware of the status  of that new version but I will have a chat with the project maintainers to figure it out soon.

**Comment by jefcolbi** [comment]
:)

**Pull request closed without merging**